.gitignore

```diff
@@ -7,4 +7,5 @@ spec/dummy/log/*.log
 spec/dummy/tmp/
 .rubocop-https*
 
-Gemfile.lock
+*/vendor/bundle/*
+*emfile.lock
```

Comment:
We don't want any of the gemfile locks to be included from appraisal either. We also don't want any of their bundled gems.

Reply:
Is this so it catches Gemfile.lock and gemfile.lock?

Reply:
Yup! For some reason appraisal's are a lower case G 😆